lib/types/reducers.js

```diff
@@ -1,9 +1,10 @@
 // @flow
 
-import {latLngBounds} from 'leaflet'
+import { latLngBounds } from 'leaflet'
 
 import UserPermissions from '../common/user/UserPermissions'
 import UserSubscriptions from '../common/user/UserSubscriptions'
+import type { PermissionType } from '../common/user/UserPermissions'
```

Comment:
Maybe insert blank line before the type imports.

Reply:
Same note as https://github.com/ibi-group/datatools-ui/pull/693#discussion_r677893400. Reverting it is undone during the pre-commit hook.